src/jit/codegenxarch.cpp

```diff
@@ -2725,6 +2746,7 @@ CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
         if (type1 != TYP_UNKNOWN)
         {
             if (structDesc.eightByteClassifications[1] == SystemVClassificationTypeIntegerReference ||
+                structDesc.eightByteClassifications[1] == SystemVClassificationTypeIntegerByReference ||
```

Comment:
Code duplication here too.  We should take this opportunity to clean up code duplication.